Source/Scene/PointCloud3DTileContent.js

```diff
@@ -305,71 +324,125 @@ define([
+                hasColors = true;
+            } else if (defined(featureTableJson.RGB)) {
+                colors = featureTable.getPropertyArray('RGB', ComponentDatatype.UNSIGNED_BYTE, 3);
+                hasColors = true;
+            } else if (defined(featureTableJson.RGB565)) {
+                colors = featureTable.getPropertyArray('RGB565', ComponentDatatype.UNSIGNED_SHORT, 1);
+                isRGB565 = true;
+                hasColors = true;
+            }
         }
 
-        // Get the colors
-        var colors;
-        var isTranslucent = false;
-        var isRGB565 = false;
-
-        if (defined(featureTableJson.RGBA)) {
-            colors = featureTable.getPropertyArray('RGBA', ComponentDatatype.UNSIGNED_BYTE, 4);
-            isTranslucent = true;
-        } else if (defined(featureTableJson.RGB)) {
-            colors = featureTable.getPropertyArray('RGB', ComponentDatatype.UNSIGNED_BYTE, 3);
-        } else if (defined(featureTableJson.RGB565)) {
-            colors = featureTable.getPropertyArray('RGB565', ComponentDatatype.UNSIGNED_SHORT, 1);
-            isRGB565 = true;
-        } else if (defined(featureTableJson.CONSTANT_RGBA)) {
-            var constantRGBA  = featureTable.getGlobalProperty('CONSTANT_RGBA', ComponentDatatype.UNSIGNED_BYTE, 4);
-            content._constantColor = Color.fromBytes(constantRGBA[0], constantRGBA[1], constantRGBA[2], constantRGBA[3], content._constantColor);
-        } else {
-            // Use a default constant color
-            content._constantColor = Color.clone(Color.DARKGRAY, content._constantColor);
+        if (!hasNormals) {
+            if (defined(featureTableJson.NORMAL)) {
+                normals = featureTable.getPropertyArray('NORMAL', ComponentDatatype.FLOAT, 3);
+                hasNormals = true;
+            } else if (defined(featureTableJson.NORMAL_OCT16P)) {
+                normals = featureTable.getPropertyArray('NORMAL_OCT16P', ComponentDatatype.UNSIGNED_BYTE, 2);
+                isOctEncoded16P = true;
+                hasNormals = true;
+            }
         }
 
-        content._isTranslucent = isTranslucent;
+        if (!hasBatchIds) {
+            if (defined(featureTableJson.BATCH_ID)) {
+                batchIds = featureTable.getPropertyArray('BATCH_ID', ComponentDatatype.UNSIGNED_SHORT, 1);
+                hasBatchIds = true;
+            }
+        }
 
-        // Get the normals
-        var normals;
-        var isOctEncoded16P = false;
+        if (!hasPositions) {
+            throw new RuntimeError('Either POSITION or POSITION_QUANTIZED must be defined.');
+        }
 
-        if (defined(featureTableJson.NORMAL)) {
-            normals = featureTable.getPropertyArray('NORMAL', ComponentDatatype.FLOAT, 3);
-        } else if (defined(featureTableJson.NORMAL_OCT16P)) {
-            normals = featureTable.getPropertyArray('NORMAL_OCT16P', ComponentDatatype.UNSIGNED_BYTE, 2);
-            isOctEncoded16P = true;
+<<<<<<< HEAD
```

Comment:
Is this a bad merge?

Reply:
Oh yes, I didn't carefully check the last merge. Fixed now.